Moolya.robot

```diff
@@ -0,0 +1,152 @@
+*** Settings ***
+Library          SeleniumLibrary
+Resource         C:\\Users\\Admin\\Documents\\GitHub\\Mentorship_Abhijit\\BrowserKeywords.robot
+Suite Setup      Open Initial Browser
+Suite Teardown   Close All Browsers
+
+# All Keywords and variables are imported from BrowserKeywords robot file
+
+# To run Press Cntrl+Shift+P  in Windows
```

Comment:
Run cmd looks like, this is common across platforms. Do not have to add OS-wise.  

**robot -d Results Moolya.robot**

Reply:
Picked that point and will implement.